compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp

```diff
@@ -32,47 +32,6 @@ struct TagDim final
 
 using TagShape = std::vector<TagDim>;
 
-int32_t flatsize(const TagShape &shape)
-{
-  int32_t size = 1;
-  for (const auto &dim : shape)
-  {
-    if (dim.value >= 1)
-      size *= dim.value;
-  }
-  return size;
-}
-
-/**
- * @brief if 'dst' has -1 valued dim, replace -1 with inferenced value
- *
- * @return  ture, if successfully replace -1 value
- *          false, otherwise
- */
-bool inference_incomplete_shape(const TagShape &src, TagShape &dst)
```

Comment:
This function inference unknown value in `reshape/shape` node. But it will be done in shape inference logic. 

So, Let's 
 * **avoid** implementing own shape inference logic here
 * use output shape of `reshape` node, instead of using `reshape/shape`